mypy_stubs/arango/database.pyi

```diff
@@ -1,2 +1,24 @@
-class StandardDatabase: ...
-class StandardCollection: ...
+# flake8: noqa
+
+from typing import Dict, List, Any
+
+from arango.collection import StandardCollection  # type: ignore
```

Comment:
This and the other `type: ignore` are because `mypy` complains that `arango.collection` doesn't have a stub. I'm not sure what would be required to address this issue without the `type: ignore`, but I figured it wasn't worth the effort, as we're not directly using `arango.collection`, it's an intermediate import.

Reply:
I'll look into this a bit and see how we would fix it. I want to put in the extra effort since we started stubbing out the bits we do use, and it seems like maybe we're losing out on some of the benefits of typechecking if we don't cross the Ts and dot the Is.

Reply:
I've actually already started looking into this a bit, and it does seem that we aren't stubbing 'StandardCollection' (the only thing we import from `arango.collection`). I'm stubbing that out now.